spring-cloud-dataflow-server-core/src/test/java/org/springframework/cloud/dataflow/server/controller/AboutControllerTests.java

```diff
@@ -159,4 +161,55 @@ public void testChecksumNoDefaults() throws Exception {
 					.andExpect(jsonPath("$.versionInfo.shell.checksumSha256").doesNotExist());
 		}
 	}
+
+	@RunWith(SpringRunner.class)
+	@SpringBootTest(classes = TestDependencies.class, properties = {"spring.cloud.dataflow.features.skipper-enabled=true"})
```

Comment:
I am not sure why does this property use `true` for skipper-enabled.

Reply:
Because we want to test that the the about controller handles the  `UnsupportedOperationException` thrown when in skipper mode.  
The TestDependecy#aboutController bean creation uses the `skipper-enabled` condition to throw to throw the exception for `streamDeployer.environmentInfo()`